orders/views.py

```diff
@@ -1,3 +1,33 @@
-from django.shortcuts import render
+from django.views import View
+from django.http  import JsonResponse
 
-# Create your views here.
+from .models     import FrameMaterial, FrameSize
+from arts.models import Art
+
+class FrameView(View):
+    def get(self, request, art_id):
+        try:
+            art = Art.objects.get(id=art_id)
+
+            frame_sizes_prices = [
+                {
+                    'size'  : frame.name,
+                    'price' : (lambda x, y, z : ((x + z) * (y + z) - (x * y)) * frame.price_rise_rate * 100)\
+                            (art.height_cm, art.width_cm, frame.thickness_cm)
```

Comment:
```suggestion
           price = lambda x, y, z, a : ((x + z) * (y + z) - (x * y)) * a* 100)
            frame_sizes_prices = [
                {
                    'size'  : frame.name,
                    'price' : price(art.height_cm, art.width_cm, frame.thickness_cm, frame.price_rise_rate)
```
람다 잘 사용해 주셨어요~~ 혹시 제가 바꾼 코드로도 동작한다면, 이편이 조금 더 가독성이 좋을 거 같습니다. 한 번 확인해 주세요.

Reply:
감사합니당 수정했어요!